noxfile.py

```diff
@@ -294,6 +294,11 @@ def prepare_release(session: nox.Session) -> None:
 
     session.log("# Generating NEWS")
     release.generate_news(session, version)
+    if sys.stdin.isatty():
+        input(
+            "Please review the NEWS file, make necessary edits, and stage them.\n"
```

Comment:
How about
```suggestion
            "Please review the NEWS file, make necessary edits, and stage them (`git add NEWS.rst`).\n"
```
?

Or the script could even run it unconditionally in this branch..

Reply:
> Or the script could even run it unconditionally in this branch..

Let's do this in a follow up!